devops/scripts/commit-type.sh

```diff
@@ -40,7 +40,7 @@ function only_allowed_files() {
 
   affected_paths=$(git show "${commit}" --pretty=oneline --name-only | tail -n +2)
   for path in $affected_paths; do
-    if [[ " ${forbidden_files[*]} " =~ " ${path} " ]]; then
+    if [[ " ${forbidden_files[*]} " =~ ${path} ]]; then
```

Comment:
I assume the linter told you to take the regex out of quotes. not sure either way. The spacing on either side is weird to begin with.

Reply:
This was the one I was most nervous about.